gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerFragment.java

```diff
@@ -103,6 +103,11 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         .flatMap(MapAdapter::getCameraMoves)
         .as(disposeOnDestroy(this))
         .subscribe(mapContainerViewModel::onCameraMove);
+    mapAdapter
+        .toObservable()
+        .flatMap(MapAdapter::getTileProviders)
+        .as(disposeOnDestroy(this))
+        .subscribe(mapContainerViewModel::onTileProvider);
```

Comment:
Naming nit: onTileProvider<event/action>

Reply:
I renamed it to `queueTileProvider` since `onTileProviderQueue` is ambiguous (not clear whether queue is a continuation of the noun or if it's a verb).

Reply:
f9a41a2650397ca3f118ba6677bb57e2affb43af